airflow/www/views.py

```diff
@@ -167,7 +164,7 @@
 logger = logging.getLogger(__name__)
 
 
-def sanitize_args(args: dict[str, str]) -> dict[str, str]:
+def sanitize_args(args: dict[str, Any]) -> dict[str, Any]:
```

Comment:
Not related, but needed to make Mypy happy. See https://github.com/pallets/flask/issues/4979